src/otx/algorithms/visual_prompting/adapters/openvino/model_wrappers/openvino_models.py

```diff
@@ -0,0 +1,207 @@
+"""Openvino Model Wrappers of OTX Visual Prompting."""
+
+# Copyright (C) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import cv2
+import numpy as np
+from openvino.model_api.adapters.inference_adapter import InferenceAdapter
+from openvino.model_api.models import ImageModel
+from openvino.model_api.models.types import (
+    BooleanValue,
+    ListValue,
+    NumericalValue,
+    StringValue,
+)
+
+from otx.api.utils.segmentation_utils import create_hard_prediction_from_soft_prediction
+
+
+class ImageEncoder(ImageModel):
+    """Image encoder class for visual prompting of openvino model wrapper."""
+
+    __model__ = "image_encoder"
+
+    def __init__(self, inference_adapter, configuration=None, preload=False):
+        super().__init__(inference_adapter, configuration, preload)
+
+    @classmethod
+    def parameters(cls) -> Dict[str, Any]:  # noqa: D102
+        parameters = super().parameters()
+        parameters.update(
+            {
+                "resize_type": StringValue(default_value="fit_to_window"),
+                "mean_values": ListValue(default_value=[123.675, 116.28, 103.53]),
+                "scale_values": ListValue(default_value=[58.395, 57.12, 57.375]),
```

Comment:
@sovrasov Should we specify these normalization values in the model? I though these values are updated when the model is exported. Please share your thought.

Reply:
I modified it to update those values into IR through mo.